modules/javascript/src/main/webapp/javascript/atmosphere.js

```diff
@@ -1763,8 +1772,6 @@
 
                 if (typeof (_request.onTransportFailure) !== 'undefined') {
                     _request.onTransportFailure(errorMessage, _request);
-                } else if (typeof (atmosphere.util.onTransportFailure) !== 'undefined') {
```

Comment:
`atmosphere.util` simply does not have a field `onTransportFailure`, so I don't see the point of having this check.